juntagrico/templates/job.html

```diff
@@ -15,13 +15,12 @@ <h3>
 {% endblock %}
 {% block content %}
     {% vocabulary "assignment_pl" as v_assignment_pl %}
-    <div class="row">
+    <div class="row pb-4">
```

Comment:
Please don not just ad spacing as you see fit, the look has to be consinsten over the whole system

Reply:
this spacing replaces a `<br>` below, so it should not change how it looks.
In some cases, which I commented below, the br was right and should be changed back. But in general I think this is the way to go.
Maybe it should be part of a separate pull request and we do that on all templates.

Reply:
i reverted the ones adding spaces which wasn't there before